functions/server.ts

```diff
@@ -107,17 +123,30 @@ async function getFunctionSecrets(): Promise<Record<string, string>> {
       return {};
     }
 
-    // Fetch all active secrets from _secrets table
-    const result = await client.queryObject<{
-      key: string;
-      value_ciphertext: string;
-    }>`
-      SELECT key, value_ciphertext 
-      FROM _secrets
-      WHERE is_active = true 
-        AND (expires_at IS NULL OR expires_at > NOW())
+    // Check if new schema exists
+    const schemaCheck = await client.queryObject<{ exists: boolean }>`
+      SELECT EXISTS (
+        SELECT 1 FROM information_schema.tables
+        WHERE table_schema = 'system' AND table_name = 'secrets'
+      ) as exists
     `;
 
+    const useNewSchema = schemaCheck.rows[0]?.exists;
```

Comment:
Maybe it is better to extract a function to check schema without duplicated codes. But it's acceptable now.